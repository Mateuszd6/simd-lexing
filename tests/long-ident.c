long long long long long long long long long long long long long long long long
long long long long long long long long long long long long long long long long
long long long long long long long long long long long long long long long long

int DonaudampfschiffahrtselektrizitatenhauptbetriebswerkbauunterbeamtengesellschaftDonaudampfschifffahrtsgesellschaftskapitanRechtsschutzversicherungsgesellschaften()
{
    return 0;
}

long long long long long long long long long long long long long long long long
long long long long long long long long long long long long long long long long
long long long long long long long long long long long long long long long long

main(){}
