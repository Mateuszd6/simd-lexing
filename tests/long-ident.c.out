../tests/long-ident.c:1:1: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:6: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:11: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:16: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:21: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:26: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:31: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:36: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:41: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:46: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:51: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:56: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:61: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:66: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:71: TOK 0 (L = 4): "long"
../tests/long-ident.c:1:76: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:1: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:6: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:11: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:16: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:21: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:26: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:31: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:36: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:41: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:46: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:51: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:56: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:61: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:66: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:71: TOK 0 (L = 4): "long"
../tests/long-ident.c:2:76: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:1: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:6: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:11: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:16: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:21: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:26: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:31: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:36: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:41: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:46: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:51: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:56: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:61: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:66: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:71: TOK 0 (L = 4): "long"
../tests/long-ident.c:3:76: TOK 0 (L = 4): "long"
../tests/long-ident.c:5:1: TOK 0 (L = 3): "int"
../tests/long-ident.c:5:5: TOK 0 (L = 160): "DonaudampfschiffahrtselektrizitatenhauptbetriebswerkbauunterbeamtengesellschaftDonaudampfschifffahrtsgesellschaftskapitanRechtsschutzversicherungsgesellschaften"
../tests/long-ident.c:5:165: TOK 1 (L = 1): "("
../tests/long-ident.c:5:166: TOK 1 (L = 1): ")"
../tests/long-ident.c:6:1: TOK 1 (L = 1): "{"
../tests/long-ident.c:7:5: TOK 0 (L = 6): "return"
../tests/long-ident.c:7:12: TOK 2 (L = 1): "0"
../tests/long-ident.c:7:13: TOK 1 (L = 1): ";"
../tests/long-ident.c:8:1: TOK 1 (L = 1): "}"
../tests/long-ident.c:10:1: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:6: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:11: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:16: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:21: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:26: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:31: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:36: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:41: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:46: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:51: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:56: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:61: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:66: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:71: TOK 0 (L = 4): "long"
../tests/long-ident.c:10:76: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:1: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:6: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:11: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:16: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:21: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:26: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:31: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:36: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:41: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:46: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:51: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:56: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:61: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:66: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:71: TOK 0 (L = 4): "long"
../tests/long-ident.c:11:76: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:1: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:6: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:11: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:16: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:21: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:26: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:31: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:36: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:41: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:46: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:51: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:56: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:61: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:66: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:71: TOK 0 (L = 4): "long"
../tests/long-ident.c:12:76: TOK 0 (L = 4): "long"
../tests/long-ident.c:14:1: TOK 0 (L = 4): "main"
../tests/long-ident.c:14:5: TOK 1 (L = 1): "("
../tests/long-ident.c:14:6: TOK 1 (L = 1): ")"
../tests/long-ident.c:14:7: TOK 1 (L = 1): "{"
../tests/long-ident.c:14:8: TOK 1 (L = 1): "}"
Parsed: 15 lines, 100 ids, 0 strings, 0 chars, 1 ints, 0 hex,   0 floats
